### PR TITLE
Features/gohan compatibility

### PR DESCRIPTION
**Pull request by brouillette**
- configurable "USE_GOHAN" setting
- primitive + "hard coded" functionality to support Gohan in the event the operator does choose to use it

### CODE REVIEW

File: bento_federation_service/search/dataset_search/dataset_search.py [hunk 1]
```diff
@@ -207,19 +219,47 @@ async def _table_search_worker(
...
-                url_args=(("query", json.dumps(data_type_queries[table_data_type])),),
+                path_fragment=path_fragment,
+                url_args=url_args,
                 method="GET",
                 auth_header=auth_header,  # Required in some cases to not get a 403
                 extra_headers=DATASET_SEARCH_HEADERS,
             )
 
+            # if "gohan" in path_fragment:
+            #     print(f"Response: {r}")
```
Comment (davidlougheed) [inline bento_federation_service/search/dataset_search/dataset_search.py:261]: if these arent being used id remove this and other comments below

File: bento_federation_service/utils.py [hunk 2]
```diff
@@ -38,7 +38,7 @@ async def peer_fetch(client: AsyncHTTPClient, peer: str, path_fragment: str, req
         urljoin(peer, path_fragment) + arg_str,
         request_timeout=TIMEOUT,
         method=method,
-        body=request_body,
+        body=request_body,validate_cert=(not CHORD_DEBUG),
```
Comment (davidlougheed) [inline bento_federation_service/utils.py:41]: add a space after comma (minor linting thing)

**Review by davidlougheed — CHANGES_REQUESTED** [review]
other than this lgtm

File: bento_federation_service/search/dataset_search/dataset_search.py (hunk 1 above)
Comment (brouillette) [reply bento_federation_service/search/dataset_search/dataset_search.py]: good catch, thanks

**Pull request merged**